File: Backend.NET/DTOs/HandoverRequest.cs
namespace FleetManagementSystem.Api.DTOs;

public class HandoverRequest
{
    public long BookingId { get; set; }
    public int? CarId { get; set; }
    public string? FuelStatus { get; set; } // "1/4", "1/2", "3/4", "Full"
    public string? Notes { get; set; }
    public DateTime? StartDate { get; set; }
    public DateTime? EndDate { get; set; }
}

File: Backend.JAVA/src/main/java/com/example/demo/dto/HandoverRequest.java
package com.example.demo.dto;

import lombok.Data;

@Data
public class HandoverRequest {
    private Long bookingId;
    private Integer carId;
    private String fuelStatus; // "1/4", "1/2", "3/4", "Full"
    private String notes;

    public Long getBookingId() {
        return bookingId;
    }

    public void setBookingId(Long bookingId) {
        this.bookingId = bookingId;
    }

    public Integer getCarId() {
        return carId;
    }

    public void setCarId(Integer carId) {
        this.carId = carId;
    }

    public String getFuelStatus() {
        return fuelStatus;
    }

    public void setFuelStatus(String fuelStatus) {
        this.fuelStatus = fuelStatus;
    }

    public String getNotes() {
        return notes;
    }

    public void setNotes(String notes) {
        this.notes = notes;
    }
}
